crates/optimism/cli/src/commands/import.rs

```diff
@@ -71,6 +71,10 @@ impl<C: ChainSpecParser<ChainSpec = OpChainSpec>> ImportOpCommand<C> {
             .sealed_header(provider_factory.last_block_number()?)?
             .expect("should have genesis");
 
+        // Reuse a single StaticFileProducer across all chunks
```

Comment:
please dont add comments like pr descriptions

these are totally useless without the pr context